demos/smart_classroom_demo_gapi/src/custom_kerenels.cpp

```diff
@@ -0,0 +1,404 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Can all these kernels be **thin** wrappers over the functionality written in the original smart classroom?

Such approach would better illustrate the idea how cheap is moving from "traditional" code to G-API execution model.

Reply:
No. Some code depends on IE. Drawing depends on KuhnMunkers ....